buildenv/jenkins/jobs/pipelines/Pipeline-Initialize.groovy

```diff
@@ -57,18 +57,12 @@ timeout(time: 10, unit: 'HOURS') {
                         userRemoteConfigs: [remoteConfigParameters]]
 
                 variableFile = load 'buildenv/jenkins/common/variables-functions.groovy'
```

Comment:
This script loads `pipeline-functions.groovy` multiple times:
variables-functions.groovy loads pipeline-functions.groovy (twice: see L203, L779)
build.groovy also loads pipeline-functions.groovy.
Some reformatting might be required in order to load the file only once. Could be done in a different PR.

Reply:
Opened an issue for myself to revisit this later.